builder/github.go

```diff
@@ -12,14 +12,17 @@ type GitHubClient interface {
 
 // NewGitHubClient returns a new GitHubClient instance. If token is an empty
 // string, then a fake client will be returned.
-func NewGitHubClient(token string) GitHubClient {
+func NewGitHubClient(token string, domain string) GitHubClient {
 	if token == "" {
 		return &nullGitHubClient{}
 	}
 
 	ts := oauth2.StaticTokenSource(&oauth2.Token{AccessToken: token})
 	tc := oauth2.NewClient(oauth2.NoContext, ts)
-	return github.NewClient(tc).Repositories
+	c := github.NewClient(tc)
+	c.BaseURL = domain
```

Comment:
builder/github.go:23: cannot use domain (type string) as type *url.URL in assignment